openprescribing/templates/entity_home_page.html

```diff
@@ -187,13 +187,15 @@ <h3>Looking for more data?</h3>
   </div>
   <div class="col-md-6">
     <div class="landing-panel">
+      {% if entity_type!="regional_team" %}
```

Comment:
I think it would make more sense to have this logic in the view and set `entity_outlier_report_url` to empty for regional teams. Then the check here becomes `{% if entity_outlier_report_url %}`.

Reply:
Agreed